mount/mount_freebsd.go

```diff
@@ -36,7 +37,7 @@ func (m *Mount) Mount(target string) error {
 	// The "syscall" and "golang.org/x/sys/unix" packages do not define a Mount
 	// function for FreeBSD, so instead we execute mount(8) and trust it to do
 	// the right thing
-	return m.mountWithHelper(target)
+	return m.mountWithHelper(filepath.Join(target, m.Target))
```

Comment:
Would it always be an error then to have a non-empty target in a mount list of 1? If this is the first mount, the directory won't exist.

Reply:
Yes it would always error.

Currently in my snapshotter, I prepare a lowerdir in an overlay mount with all the mountpoints already created before chaining a bind mount to a non-empty target. Do you think it should be the runtime's responsibility to ensure the existence of these mountpoints?